app/controllers/application_controller.rb

```diff
@@ -8,6 +8,14 @@ class ApplicationController < ActionController::Base
   protected
 
   def configure_permitted_parameters
-    devise_parameter_sanitizer.for(:sign_up) << :username
+    devise_parameter_sanitizer.for(:sign_up) {
+      |u| u.permit(:username, :email, :password, :password_confirmation, :remember_me)
```

Comment:
Line is too long. [86/80]<br>Block argument expression is not on the same line as the block start.